js/index.js

```diff
@@ -1,23 +1,35 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+  //console.log('Calculating subtotal, yey!');
```

Comment:
I also recommend that you remove commented out code as it serves no purpose as well